src/wallet/wallet.h

```diff
@@ -579,11 +579,10 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
         SetNull();
     }
 
-    CWallet(const std::string& strWalletFileIn)
+    CWallet(const std::string& strWalletFileIn) : strWalletFile(strWalletFileIn)
```

Comment:
I can make that change, thats not an issue ;)

However, any ideas on the test failing?